lms/djangoapps/certificates/templates/certificates/edx_ace/passedcourse/email/body.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'schedules/edx_ace/common/base_body.html' %}
+{% load i18n %}
+
+{% block preview_text %}
+    {% blocktrans trimmed %}
+        Congratulations on passing {{course_name}}!
```

Comment:
Someone from product should probably review all of the text in these emails!

Reply:
+1